src/containers/ArticlePage/components/PanelTitle.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import isEqual from "lodash/fp/isEqual";
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "@emotion/styled";
+import { colors } from "@ndla/core";
+import { IArticle } from "@ndla/types-backend/draft-api";
+import { Text } from "@ndla/typography";
+import { PUBLISHED } from "../../../constants";
+
+const contentPanelChanges = (
+  current: IArticle | undefined,
+  lastPublished: IArticle | undefined,
+  fields: (keyof IArticle)[],
+): boolean => {
+  if (current === undefined || lastPublished === undefined) return false;
+  for (const field of fields) {
+    if (!isEqual(current[field], lastPublished[field])) return true;
+  }
+  return false;
+};
+
+const StyledText = styled(Text)`
+  color: ${colors.brand.grey};
+`;
+
+interface PanelTitleProps {
+  title: string;
+  article: IArticle | undefined;
+  articleHistory: IArticle[] | undefined;
+  fieldsToIndicatedChangesFor: (keyof IArticle)[];
+}
+
+const PanelTitle = ({ title, fieldsToIndicatedChangesFor, article, articleHistory }: PanelTitleProps) => {
+  const { t } = useTranslation();
+  const hasChanges = useMemo(() => {
+    const lastPublishedVersion = articleHistory?.find((a) => a.status.current === PUBLISHED);
+    return contentPanelChanges(article, lastPublishedVersion, fieldsToIndicatedChangesFor);
+  }, [article, articleHistory, fieldsToIndicatedChangesFor]);
```

Comment:
Hvis du skal ha `fieldsToIndicatedChangesFor` i dependency array her så burde den memoizes i parent, hvis ikke vil dette gjøre at denne blir kjørt på hver render

Reply:
Deet gir mening, i fix!

Finnes det noen måte å tvinge dette på fra inni komponenten? Ser for meg at det er lett å ta i bruk en komponent som dette og ikke tenke over sånt.

Reply:
Trenger vi fieldsToIndicatedChangesFor i dependency array? Vil vel aldri endre seg? Kreves isåfall en litt stygg eslint-disable da

Reply:
Hmm, _trenger_ vel ikke, men syns kanskje eslint-disable kanskje har mer potensiale for trøbbel ved en eventuell refaktorering?

Reply:
Ja, enig i at det er en uting å bruke 😓

Reply:
Daa tror jeg at jeg lar det stå sånn, med mindre du kommer med en kommentar til der du sier at jeg skal endre det :star2: